label/models/label_print.py

```diff
@@ -105,8 +105,9 @@ class IrModelFields(models.Model):
 
     @api.model
     def name_search(self, name='', args=None, operator='ilike', limit=None):
-        data = self._context['model_list']
-        args.append(('model', 'in', eval(data)))
+        if 'model_list' in self._context:
```

Comment:
Try with self._context.keys()?

Reply:
I think is better with 'model_list' in self._context because doesn't depends any python version

Reply:
.keys() is best

Reply:
Ok! Fine! Please see the updated patch